dev/generate_bom.py

```diff
@@ -289,14 +289,25 @@ def __publish_config(self, component, profile_path):
     """
     for profile in os.listdir(profile_path):
       full_profile = os.path.join(profile_path, profile)
+      profile_path = full_profile
```

Comment:
I think `profile_path` and `full_profile` will always point to the individual profile files in `halconfig` the way this is invoked now. Seems like you can get rid of the loop and just tar up the `profile_path`, which is a directory.